shared/chat/inbox/row/small-team/index.js

```diff
@@ -39,23 +39,51 @@ type Props = {
   youNeedToRekey: boolean,
 }
 
+type State = {
+  isHovered: boolean,
+}
+
 const SmallTeamBox = isMobile
-  ? Box
+  ? ClickableBox
   : glamorous(Box)({
       '& .small-team-gear': {display: 'none'},
       ':hover .small-team-gear': {display: 'unset'},
       ':hover .small-team-timestamp': {display: 'none'},
     })
 
-class SmallTeam extends React.PureComponent<Props> {
+class SmallTeam extends React.PureComponent<Props, State> {
+  constructor(props: Props) {
```

Comment:
Perhaps you can get away with `state = { isHovered: false }` for some shorthand state initialization.